notebooks/03_evaluate/evaluation.ipynb

```diff
@@ -121,7 +121,7 @@
    "source": [
```

Comment:
<p>I think there's still a type in the first sentence here, maybe a better version would be:</p><p>"For comparison, a similar process is used with a threshold value of 3 to create a more balanced dataset."</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/743/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Reply:
Great suggestion. I used it.